templates/ovncontroller/bin/stop-vswitchd.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -x
+
+ovs_folder=/var/lib/openvswitch
+
+if [ -f $ovs_folder/flows ]; then
```

Comment:
```
  files=(...)
  for i in $files; do
    if [ -f $file ]; then
      rm -f ...
    fi
  done
```

Reply:
Since I removed the bridges file, I think a for is not needed anymore